### PR TITLE
chore(wallet): add images to select address page

### LINKED ISSUE #18036: Update Wallet Illustrations on send page
The send page has placeholder boxes in place of illustrations. 
Update the illustrations for Wallet send page empty states.

You will find them in the wallet send flow 🙂
https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=689%3A192786&mode=design&t=M4ByOcH9p8vhjnmO-1

### PR DESCRIPTION
**Pull request by J-Son89**
fixes: https://github.com/status-im/status-mobile/issues/18036

This adds the images to the select address page in the new wallet flow. 

also adjusted the copy slightly to match with the designs.

Some small refactors within the resources file but this should not affect any changes to the code.
Pages I made changes on include:
Chat home page, communities home page, Quo Empty State component preview

Video demo'ing all of these pages.

https://github.com/status-im/status-mobile/assets/22799766/efc43988-0179-44d6-acbd-5a683f07d33b

### CODE REVIEW

**Review by smohamedjavid — APPROVED** [review]
Nice!

File: src/status_im2/common/resources.cljs [hunk 1]
```diff
@@ -22,43 +22,29 @@
    :onboarding-illustration       (js/require "../resources/images/ui2/onboarding_illustration.png")
    :qr-code                       (js/require "../resources/images/ui2/qr-code.png")
    :keycard-logo                  (js/require "../resources/images/ui2/keycard-logo.png")
-   :keycard-chip-light            (js/require "../resources/images/ui2/keycard-chip-light.png")
-   :keycard-chip-dark             (js/require "../resources/images/ui2/keycard-chip-dark.png")
+
```
Comment (OmarBasem) [inline src/status_im2/common/resources.cljs]: unnecessary new line?

File: src/status_im2/common/resources.cljs [hunk 2]
```diff
@@ -22,43 +22,29 @@
...
-   :no-communities-dark           (js/require "../resources/images/ui2/no-communities-dark.png")
-   :no-pending-communities-light  (js/require "../resources/images/ui2/no-pending-communities-light.png")
-   :no-pending-communities-dark   (js/require "../resources/images/ui2/no-pending-communities-dark.png")
-   :no-opened-communities-light   (js/require "../resources/images/ui2/no-opened-communities-light.png")
-   :no-opened-communities-dark    (js/require "../resources/images/ui2/no-opened-communities-dark.png")
-   :no-contacts-to-invite-light   (js/require "../resources/images/ui2/no-contacts-to-invite-light.png")
-   :no-contacts-to-invite-dark    (js/require "../resources/images/ui2/no-contacts-to-invite-dark.png")
-   :no-notifications-light        (js/require "../resources/images/ui2/no-notifications-light.png")
-   :no-notifications-dark         (js/require "../resources/images/ui2/no-notifications-dark.png")
+
```
Comment (OmarBasem) [inline src/status_im2/common/resources.cljs]: unnecessary new line?

**Review by OmarBasem — APPROVED** [review]

File: src/status_im2/common/resources.cljs (hunk 1 above)
Comment (J-Son89) [reply src/status_im2/common/resources.cljs]: thanks, will adjust these!

**Review by ulisesmac — APPROVED** [review]
Looks good

**Review by vkjr — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
Hi @J-Son89 Thank you for PR. No issues from my side.

@Francesca-G could you please take a look current PR?

**Review by Francesca-G — COMMENTED** [review]
There's an issue with the dark theme: it looks good in the Wallet empty states but in the Messages and Communities home the image stays black while the text has the correct color, example here:

IMG_0479


I've also noticed that the Permission tab is missing from the Wallet account overview, is this yet to be implemented? Just checking to make sure :)
It should be between "Activity" and "dApps":

Screenshot 2023-12-14 alle 13 09 15

**Comment by J-Son89** [comment]
good spot @Francesca-G - will address that!

Permissions was removed for the moment on request of @John-44. I will add a note in the designs about this.

**Comment by J-Son89** [comment]
@VolodLytvynenko - I rebased and tried to reproduce the theme issue that @Francesca-G mentioned. I can not reproduce this, what should I do here? 🤔

**Comment by VolodLytvynenko** [comment]
hey @J-Son89 here the steps:
1. Go to the appearance settings and switch the theme (changing from dark mode to light mode or vice versa is mandatory)
2. Go to list of 1-1 chats or list of communities

Actual result:
- the status bar (containing battery and time indicators) merges with the Status app backround
- The images are merged on 1-1 list chat and on the community list

https://github.com/status-im/status-mobile/assets/52490791/b4541782-a196-46b5-9627-1f2327ddb486


@J-Son89 @Francesca-G This issue occurs after changing the theme in the appearance settings and resolved after relogin. To my knowledge, a separate issue has already been created for this and will be addressed separately. It's also present in the nightly build and the last release, and it's not related with the current PR.

I suggest merging this PR, and I'll double-check if the issue has already been reported. If not, I'll report it.

**Comment by VolodLytvynenko** [comment]
> There's an issue with the dark theme: it looks good in the Wallet empty states but in the Messages and Communities home the image stays black while the text has the correct color, example here:

Hi @Francesca-G this issue will be fixed separately here https://github.com/status-im/status-mobile/issues/17636. The images are resolved on dark/ light theme after relogin

**Comment by J-Son89** [comment]
@VolodLytvynenko - yeah I checked and this issue with theme is in develop too. I think it's not the same as the status bar theme issue but agree it's better to handle separately.

**Pull request merged**